pkgs/development/compilers/graalvm/default.nix

```diff
@@ -172,13 +172,13 @@ let
 in rec {
 
   mx = stdenv.mkDerivation rec {
-    version = "5.224.10";
+    version = "5.223.0";
```

Comment:
this is actually a downgrade to use the exact version which 19.1.1 uses
https://github.com/oracle/graal/compare/vm-19.1.0...release/graal-vm/19.1